client/src/api/block_builder/input_selection/core/mod.rs

```diff
@@ -131,7 +133,7 @@ impl InputSelection {
 
         // Gets requirements from outputs.
         // TODO this may re-evaluate outputs added by inputs
```

Comment:
Can this TODO be removed now?

Reply:
Not sure yet, I need to investigate about the order in which things are evaluated. It's in my list, will do later this week.